pkg/apis/core/validation/validation_test.go

```diff
@@ -5848,9 +5848,14 @@ func TestValidateEphemeralContainers(t *testing.T) {
 					TerminationMessagePath:   "/dev/termination-log",
```

Comment:
Can you `s/Whitelisted/allowed/` on line 5825? 
https://github.com/kubernetes/community/blob/master/wg-naming/recommendations/002-blacklist-whitelist-allowlist-denylist.md

Same below in the failure cases (lines 5931+)